web/src/emoji.ts

```diff
@@ -295,7 +295,7 @@ export function get_emoji_details_by_name(emoji_name: string): EmojiRenderingDet
         const emoji_code_info = active_realm_emojis.get(emoji_name)!;
         return {
             emoji_name,
-            emoji_code: emoji_code_info.id,
+            emoji_code: emoji_code_info.id.toString(),
```

Comment:
@andersk does this seem okay?

Reply:
No, `id` is already a `string` at runtime; its annotation as `number` in the `RealmEmoji` and `ServerEmoji` types is wrong.

Reply:
Okay, changed those